Build error when disabling easylogging

SDK Version 2.54.1

If I disable easylogging i get the following build error using gcc10

```
src/librealsense/src/terminal-parser.cpp: In member function ‘std::vector<unsigned char> librealsense::terminal_parser::parse_command(const string&) const’:
src/librealsense/src/terminal-parser.cpp:37:13: error: ‘cout’ was not declared in this scope
   37 |             cout << hex << fixed << setfill('0') << setw(2) << (int)b << " ";
      |             ^~~~
src/librealsense/src/terminal-parser.cpp:5:1: note: ‘std::cout’ is defined in header ‘<iostream>’; did you forget to ‘#include <iostream>’?
```
maybe specifically include **<iostream>** in terminal-parser.cpp ?

--- a/.github/workflows/buildsCI.yaml
+++ b/.github/workflows/buildsCI.yaml
@@ -21,7 +21,7 @@ jobs:
 
 
   #--------------------------------------------------------------------------------
-  Win_SH_EX_CfU:  # Windows, shared, with Examples & Tools, and Check for Updates
+  Win_SH_EX_CfU:  # Windows, shared, with Examples & Tools, no EasyLogging and Check for Updates
     runs-on: windows-2019
     timeout-minutes: 60
     steps:
@@ -53,7 +53,7 @@ jobs:
         cd ${{env.WIN_BUILD_DIR}}
         pwd
         ls        
-        cmake ${LRS_SRC_DIR} -G "Visual Studio 16 2019" -DBUILD_SHARED_LIBS=true -DBUILD_EXAMPLES=true -DBUILD_TOOLS=true -DCHECK_FOR_UPDATES=true
+        cmake ${LRS_SRC_DIR} -G "Visual Studio 16 2019" -DBUILD_SHARED_LIBS=true -DBUILD_EXAMPLES=true -DBUILD_TOOLS=true -DCHECK_FOR_UPDATES=true -DBUILD_EASYLOGGINGPP=false
 
     - name: Build
       # Build your program with the given configuration

--- a/.github/workflows/buildsCI.yaml
+++ b/.github/workflows/buildsCI.yaml
@@ -21,7 +21,7 @@ jobs:
 
 
   #--------------------------------------------------------------------------------
-  Win_SH_EX_CfU:  # Windows, shared, with Examples & Tools, no EasyLogging and Check for Updates
+  Win_SH_EX_CfU:  # Windows, shared, with Examples & Tools, and Check for Updates
     runs-on: windows-2019
     timeout-minutes: 60
     steps:
@@ -53,7 +53,49 @@ jobs:
         cd ${{env.WIN_BUILD_DIR}}
         pwd
         ls        
-        cmake ${LRS_SRC_DIR} -G "Visual Studio 16 2019" -DBUILD_SHARED_LIBS=true -DBUILD_EXAMPLES=true -DBUILD_TOOLS=true -DCHECK_FOR_UPDATES=true -DBUILD_EASYLOGGINGPP=false
+        cmake ${LRS_SRC_DIR} -G "Visual Studio 16 2019" -DBUILD_SHARED_LIBS=true -DBUILD_EXAMPLES=true -DBUILD_TOOLS=true -DCHECK_FOR_UPDATES=true
+
+    - name: Build
+      # Build your program with the given configuration
+      shell: bash
+      run: |
+        cd ${{env.WIN_BUILD_DIR}}
+        cmake --build . --config ${{env.LRS_RUN_CONFIG}} -- -m
+
+  #--------------------------------------------------------------------------------
+  Win_SH_EX_No_Logs:  # Windows, shared, with Examples & Tools, no EasyLogging and no Check for Updates
+    runs-on: windows-2019
+    timeout-minutes: 60
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Enable Long Paths
+      shell: powershell
+      run: |
+       New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
+    
+    - name: Check_API
+      shell: bash
+      run: |
+        cd scripts
+        ./api_check.sh
+        cd ..
+
+    - name: PreBuild
+      shell: bash
+      run: |
+       mkdir ${{env.WIN_BUILD_DIR}}
+    
+    - name: Configure CMake
+      # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+      # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+      shell: bash
+      run: |        
+        LRS_SRC_DIR=$(pwd)
+        cd ${{env.WIN_BUILD_DIR}}
+        pwd
+        ls        
+        cmake ${LRS_SRC_DIR} -G "Visual Studio 16 2019" -DBUILD_SHARED_LIBS=true -DBUILD_EXAMPLES=true -DBUILD_TOOLS=true -DCHECK_FOR_UPDATES=false -DBUILD_EASYLOGGINGPP=false
 
     - name: Build
       # Build your program with the given configuration
@@ -61,8 +103,7 @@ jobs:
       run: |
         cd ${{env.WIN_BUILD_DIR}}
         cmake --build . --config ${{env.LRS_RUN_CONFIG}} -- -m    
-
-
+        
   #--------------------------------------------------------------------------------
   Win_ST_Py_CI:  # Windows, Static, Python, Tools, libCI with executables
     runs-on: windows-2019

--- a/src/proc/formats-converter.h
+++ b/src/proc/formats-converter.h
@@ -7,6 +7,7 @@
 
 #include <vector>
 #include <unordered_set>
+#include <unordered_map>
 #include <memory>
 
 namespace librealsense

--- a/src/terminal-parser.cpp
+++ b/src/terminal-parser.cpp
@@ -32,12 +32,6 @@ namespace librealsense
 
         auto raw_data = build_raw_command_data(command, params);
 
-        for (auto b : raw_data)
-        {
-            cout << hex << fixed << setfill('0') << setw(2) << (int)b << " ";
-        }
-        cout << endl;
-
         return raw_data;
     }
 

